packages/react-html/src/serializer.tsx

```diff
@@ -18,10 +18,7 @@ export function useSerialized<T>(
   id: string,
 ): [T | undefined, React.ComponentType<SerializeProps<T>>] {
   const manager = React.useContext(HtmlContext);
-  const data = React.useMemo(
```

Comment:
This memoization prevents getting the latest value and it doesn't really optimize anything since a `Map` access time is probably inconsequential.